src/System.Management.Automation/commands/utility/FormatAndOutput/common/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -534,16 +534,16 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Diagnostics_Proc
             yield return new FormatViewDefinition("process",
                 TableControl.Create()
                     .AddHeader(Alignment.Right, label: "NPM(K)", width: 7)
-                    .AddHeader(Alignment.Right, label: "PM(K)", width: 8)
-                    .AddHeader(Alignment.Right, label: "WS(K)", width: 10)
+                    .AddHeader(Alignment.Right, label: "PM(M)", width: 8)
+                    .AddHeader(Alignment.Right, label: "WS(M)", width: 10)
                     .AddHeader(Alignment.Right, label: "CPU(s)", width: 10)
                     .AddHeader(Alignment.Right, width: 6)
                     .AddHeader(Alignment.Right, width: 3)
                     .AddHeader()
                     .StartRowDefinition()
                         .AddScriptBlockColumn("[long]($_.NPM / 1024)")
-                        .AddScriptBlockColumn("[long]($_.PM / 1024)")
-                        .AddScriptBlockColumn("[long]($_.WS / 1024)")
+                        .AddScriptBlockColumn("\"{0:N3}\" -f [float]($_.PM / (1024 * 1024))")
```

Comment:
I would use `1mb` instead of `1024 * 1024` because PowerShell does not implement constant folding.
I think I would also use `:N2` instead of `:N3`, I don't think we require that much precision and helps avoid exceeding the maximum column width now that we see a `.` that wasn't there before.